src/Compute/Compute/VirtualMachine/Operation/NewAzureVMCommand.cs

```diff
@@ -1019,6 +1019,20 @@ public void DefaultExecuteCmdlet()
                         }
                     }
                 }
+                else if (this.VM.StorageProfile?.ImageReference?.SharedGalleryImageId != null) {
```

Comment:
for these else if blocks, include the if jobPresent in here with an &&. They're simple enough.

Reply:
I wanted to follow the style of the other if in that condition

Reply:
if AsJobPresent prints a warning message, if true, it doesn't print, so it has to stay this way